File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int maxScore = 0;
        int sum = 0;
        int[] nums = new int[n];
        for (int i = 0; i < n; i++) {
            nums[i] = sc.nextInt();
            maxScore = Math.max(maxScore, nums[i]);
            sum += nums[i];
        }

        System.out.println(sum * 100.0 / maxScore / n);
    }
}

File: 백준/Bronze/1546. 평균/평균.kt
fun main() {
    val n = readLine()!!.toInt()
    var maxScore = 0
    var sum = 0
    val nums = readLine()!!.split(" ")
    for (i in 0 until n) {
        val score = nums[i].toInt()
        if (score > maxScore) maxScore = score
        sum += score
    }
    
    println(sum * 100.0 / maxScore / n)
}